vignettes/version_policy.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "R version support"
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  \usepackage[utf8]{inputenc}
+  %\VignetteIndexEntry{R version support}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+R evolves every year. Each update brings many bug fixes and performance 
+improvements, but also sometimes changes the default value of some parameters
+(such as switching from `stringsAsFactors = TRUE` to `FALSE` in R 4.0.0).
+
+While there is benefit in supporting older R versions, doing so adds 
+maintenance burden, especially in a large ecosystem like **easystats** that
+contains 10 packages that are seamlessly supposed to work together. Therefore,
+**easystats** is supported (meaning each package is thoroughly tested) on the
+**current R** version, the future one (**R-devel**) and the **previous four**
+versions. This is the same as the [version policy of `tidyverse`
+ecosystem](https://www.tidyverse.org/blog/2019/04/r-version-support/).
+
+Supposing R updates keep happening once a year in Spring, this table shows the 
+corresponding latest R version and the oldest supported version:
+
+| Year | Latest R version | Oldest supported R version |
+|------|------------------|----------------------------|
+| 2024 | 4.4              | 4.0                        |
+| 2025 | 4.5              | 4.1                        |
+| 2026 | 4.6              | 4.2                        |
+| 2027 | 4.7              | 4.3                        |
+| 2028 | 4.8              | 4.4                        |
+
+## Implications
+
+The implications are simple:
+
+* if you are a user and your R version is below the oldest supported one, you
```

Comment:
Nit: Since both of these unordered list items are no longer standalone clauses, can you also capitalize the first letter in the sentence.